README.md

```diff
@@ -1,131 +1,165 @@
+<img src="./screenshot.png" alt="usehooks-ts banner" align="center" />
+
+<br />
+
 <div align="center">
 <h1>useHooks(🔥).ts</h1>
 
-React hook library, ready to use, written in Typescript.
+<div>React hook library, ready to use, written in Typescript.</div>
 
-&mdash; [usehooks-typescript.com](https://usehooks-typescript.com/) &mdash;
-
-**README Sections:** [About](#-about) - [Hook Summary](#-summary) - [Installation](#-installation) - [Made with](#-made-with) - [Contributing](#-contributing) - [License](#-license)
+<br />
 
 <!-- Badges -->
 
 [![Netlify Status](https://api.netlify.com/api/v1/badges/f1f0f5a4-8207-499b-b912-d99acb04176e/deploy-status)](https://app.netlify.com/sites/usehooks-ts/deploys)
 [![Open Source? Yes!](https://badgen.net/badge/Open%20Source%20%3F/Yes%21/blue?icon=github)](https://github.com/juliencrn/usehooks.ts/issues)
 [![Maintained](https://badgen.net/badge/Maintained%20%3F/Yes%21/blue?icon=github)](https://github.com/juliencrn/usehooks.ts/issues)
 [![License](https://badgen.net/badge/License/MIT/blue)](https://github.com/juliencrn/usehooks.ts/blob/master/LICENSE)
+[![lerna](https://img.shields.io/badge/maintained%20with-lerna-cc00ff.svg)](https://lerna.js.org/)
+![npm bundle size](https://img.shields.io/bundlephobia/minzip/usehooks-ts)
+![npm](https://img.shields.io/npm/v/usehooks-ts)<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+[![All Contributors](https://img.shields.io/badge/all_contributors-21-orange.svg?style=flat-square)](#contributors-)
+
+<!-- ALL-CONTRIBUTORS-BADGE:END -->
+
+<div align="center">
+  <sub>Created by <a href="https://github.com/juliencrn">Julien Caron</a> and maintained with ❤️ by an amazing <a href="#contributors">team of developers</a>.</sub>
+</div>
 
 </div>
 
+<br />
+
 ## 🤔 About
 
-This is the repository for [usehooks.ts](https://usehooks-typescript.com/), a [Gatsby](https://www.gatsbyjs.org) powered blog hosted with Github & netlify that publishes easy to understand React Hook code snippets.
+Initially, useHooks.ts was a [Gatsby](https://www.gatsbyjs.org) powered blog hosted with Github & netlify that publishes easy to understand React Hook code snippets.
 
-UseHooks.ts has been strongly inspired by [usehooks.com](https://usehooks.com), thanks to him. My objective here is to publish my hooks in Typescript in order to find them quickly
+But now, it's a monorepo containing:
+
+- A static website used as hooks documentation ([Link](https://usehooks-typescript.com/)).
+- A NPM packages containing the hooks library ([Link](https://www.npmjs.com/package/usehooks-ts)).
```

Comment:
*An (it's weird, but it's the sound, not the letter that counts, hence "An En-pi-em" vs "A En-pi-em" - NPM's first sounds is a vowel)
https://english.stackexchange.com/questions/105116/is-it-a-user-or-an-user

*package

Reply:
I also asked myself the question haha. 

Thanks for the feedback.